### PR TITLE
CY-2914 Add the required changes for syncthing to work with supervisord

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

File: cfy_manager/components/rabbitmq/scripts/start_rabbitmq_server.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# call "rabbitmqctl shutdown" when exiting
+trap "{ echo Stopping rabbitmq; /usr/sbin/rabbitmqctl shutdown; exit 0; }" EXIT
+
+echo "Starting rabbitmq"
+/usr/sbin/rabbitmq-server
```
Comment (geokala) [inline cfy_manager/components/rabbitmq/scripts/start_rabbitmq_server.sh:9]: Minor nitpick: It's nicer to have a newline on the end of the file.

Comment (mabuaisha) [reply cfy_manager/components/rabbitmq/scripts/start_rabbitmq_server.sh:9]: Done

File: cfy_manager/components/rabbitmq/rabbitmq.py [hunk 2]
```diff
@@ -486,10 +493,24 @@ def _set_config(self):
                 }
             }
 
+    def _configure_rabbitmq_wrapper_script(self):
+        deploy(
+            join(
+                SCRIPS_PATH,
```
[the remark this answers is not in the document]
Comment (mabuaisha) [reply cfy_manager/components/rabbitmq/rabbitmq.py:499]: Done

File: cfy_manager/components/postgresql_server/postgresql_server.py [hunk 3]
```diff
@@ -547,7 +547,14 @@ def _configure_cluster(self):
                     service_type=self.service_type
                 ))
         common.sudo(['mv', '-T', tmp_path, '/etc/rsyslog.d/43-etcd.conf'])
-        # Restart the rsyslog
+        # Configure the rsyslog for supervisord
```
[the remark this answers is not in the document]
Comment (mabuaisha) [reply cfy_manager/components/postgresql_server/postgresql_server.py:550]: Done

**Pull request merged**